packages/core/admin/admin/src/translations/fr.json

```diff
@@ -588,6 +588,38 @@
   "content-manager.success.record.save": "Sauvegardé",
   "content-manager.success.record.unpublish": "Publication annulée",
   "content-manager.utils.data-loaded": "{number, plural, =1 {L'entrée a été chargée} other {Les entrées on été chargées} avec  succès",
+  "content-manager.apiError.This attribute must be unique": "Le champ {field} doit être unique",
+  "form.button.continue": "Continuer",
+  "global.search": "Rechercher",
+  "global.actions": "Actions",
+  "global.back": "Retour",
+  "global.cancel": "Annuler",
+  "global.change-password": "Modifier le mot de passe",
+  "global.content-manager": "Gestion du contenu",
+  "global.continue": "Continuer",
+  "global.delete": "Supprimer",
+  "global.delete-target": "Supprimer {target}",
+  "global.description": "Description",
+  "global.details": "Détails",
+  "global.disabled": "Désactivé",
+  "global.documentation": "Documentation",
+  "global.enabled": "Activé",
+  "global.finish": "Terminer",
+  "global.marketplace": "Marketplace",
+  "global.name": "Nom",
+  "global.none": "Aucun",
+  "global.password": "Mot de passe",
+  "global.plugins": "Plugins",
+  "global.profile": "Profil",
+  "global.reset-password": "Réinitialiser le mot de passe",
+  "global.roles": "Rôles",
+  "global.save": "Enregistrer",
+  "global.see-more": "Voir plus",
+  "global.select": "Séléctionner",
```

Comment:
oh sorry I missed this one before approving, could you fix it please @Zharkan? ('_Selectionner_')

Reply:
My bad, fixed it !